### PR TITLE
#43 hide a field / added HiddenWidget

### PR DESCRIPTION
**Pull request by HelloLudger**
Added HiddenWidget and hide HiddenFields by adding `display: none` to Wrapper within SchemaFields.js.

Sorry, no tests. Couldn't get the tests to run, in addition to my 0-knowledge of testing.

### CODE REVIEW

File: src/components/fields/SchemaField.js [hunk 1]
```diff
@@ -81,6 +83,7 @@ function SchemaField(props) {
   return (
     <Wrapper
       label={schema.title || name}
+      hidden={(uiSchema["ui:widget"] === "hidden" ? true : false)}
```
Comment (n1k0) [inline src/components/fields/SchemaField.js]: Nit: `uiSchema["ui:widget"] === "hidden"` is good enough

File: src/components/widgets/HiddenWidget.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  required,
+  onChange
+}) {
+  return (
+    <input type="hidden"
+      value={value}
+      defaultValue={defaultValue}
+      placeholder={placeholder}
+      required={required}
+      onChange={(event) => onChange(event.target.value)} />
```
Comment (n1k0) [inline src/components/widgets/HiddenWidget.js]: Is this input value supposed to change ever?

**Comment by n1k0** [comment]
We definitely want tests and documentation updated here. What prevents you running them? What error are you getting?

File: src/components/widgets/HiddenWidget.js (hunk 2 above)
Comment (HelloLudger) [inline src/components/widgets/HiddenWidget.js]: good question... I thought about it. Maybe in combination with [_Add computed fields ? #45_](https://github.com/mozilla-services/react-jsonschema-form/issues/45).
But I think it's safe to say that the placeholder won't be needed.

**Comment by HelloLudger** [comment]
Maybe the problem is that I use Windows. I don't know. When I start `npm test` the error says that the command "NODEv_ENV" is not a viable command.

```
C:\Projekte\react-jsonschema-form>npm test

> react-jsonschema-form@0.12.0 test C:\Projekte\react-jsonschema-form
> NODE_ENV=test mocha --compilers js:babel/register --recursive --require ./test/setup-jsdom.js $(find test -name '*_test.js')

Der Befehl "NODE_ENV" ist entweder falsch geschrieben oder
konnte nicht gefunden werden.
npm ERR! Test failed.  See above for more details.
```

**Comment by n1k0** [comment]
Superseded by #111.

**Pull request closed without merging**